GitUpKit/Views/Base.lproj/GICommitSplitterViewController.xib

```diff
@@ -46,7 +46,7 @@
                             <rect key="frame" x="127" y="8" width="855" height="16"/>
                             <autoresizingMask key="autoresizingMask" widthSizable="YES" flexibleMinY="YES"/>
                             <textFieldCell key="cell" controlSize="small" lineBreakMode="truncatingTail" allowsUndo="NO" sendsActionOnEndEditing="YES" alignment="left" title="&lt;TITLE&gt;" id="2Ta-mo-RyA">
-                                <font key="font" metaFont="controlContent"/>
+                                <font key="font" metaFont="cellTitle"/>
```

Comment:
Just glancing at this diff, does this result in a different font?

Reply:
I was thinking that too. Asked a co-worker and he said it's probably just related to me using a new version of XCode. I couldn't identify any changed fonts.

Reply:
Ya, I didn't notice anything that looked wrong when looking it over.